### DISCUSSION TITLE
Installed from free v3 to AGPL v4 - now it acts like a single user website?

### DISCUSSION THREAD

**Discussion by accessiblepixel** [chevereto/chevereto#23] [Q&A]
I'm not sure if this is right, but upgrading from free to the AGPL v4 code from github and it only shows my profile, not any of the other users on the instance?

It doesn't match the screenshots that appear in the repo and trying to even view other users registered on the system is "locked" behind a pro license?

Is all accessible functionality now locked behind the 67$ license or have I just encountered a bug?

Kind regards,

**Comment by rodber** [comment]
The project at [rodber/chevereto-free](https://github.com/rodber/chevereto-free) doesn't share functionality with the project I run here. I wrote about this in [this post](https://rodolfoberrios.com/2022/12/01/chevereto-goes-free/).

**Comment by accessiblepixel** [comment]
Ah, so "Yes" appears to be the answer, not a bug. I can understand wanting to make money especially from those that could afford it and run big sites, but it's not a piece of software I can afford so that a couple of us can use it - I'll have to look for an alternative.

Good luck with everything, I know you got rid of the discord at some point so hope all goes well!

Kind regards

**Comment by rodber** [comment]
> I know you got rid of the discord at some point so hope all goes well!

https://discord.com/invite/4SYkyrTGuj
